code/modules/martial_arts/combos/judo/fatality.dm

```diff
@@ -0,0 +1,19 @@
+/datum/martial_combo/judo/fatailty
+	name = "Meele Finisher"
```

Comment:
The term melee finisher was used due to my plan to add some combos that used the disabler in the offhand, but I felt that would be adding too much complexity to the martial art (as cool as it would be). Renaming this combo to "Wheel Throw" for clarity.